common/components/componentsBySection/FindProjects/ProjectCard.jsx

```diff
@@ -169,6 +175,32 @@ class ProjectCard extends React.PureComponent<Props, State> {
       </div>
     );
   }
+
+  _renderCardOperation(): ?React$Node {
+    const cardOperation: CardOperation = this.props.project.cardOperation;
+    if (cardOperation) {
+      let buttonConfig: Dictionary<any> = {};
+      if (cardOperation.operation) {
+        buttonConfig = {
+          onClick: () => cardOperation.operation(),
+        };
+      } else {
+        buttonConfig = { href: cardOperation.url };
+      }
+      return (
+        <React.Fragment>
+          <Button
+            variant="primary"
```

Comment:
For the design as given, toggling between `variant="primary"` and then `variant="outline-secondary"` matches the requested look.